src/TableView.jl

```diff
@@ -1,119 +1,155 @@
 module TableView
-
-using WebIO
-using JSExpr
-using JuliaDB
-using DataValues
-
-import JuliaDB: DNDSparse, DNextTable, NextTable
-
-function JuliaDB.subtable(t::DNextTable, r)
-    table(collect(rows(t)[r]), pkey=t.pkey)
+using Tables, TableTraits, IteratorInterfaceExtensions
+using WebIO, JSExpr, JSON, Dates
+using Observables: @map
+using AssetRegistry
+
+const ag_grid_imports = []
+
+function __init__()
+    empty!(ag_grid_imports)
+    for f in ["ag-grid.js", "ag-grid.css", "ag-grid-light.css", "ag-grid-dark.css"]
+        path = normpath(joinpath(@__DIR__, "..", "deps", "ag-grid", f))
+        AssetRegistry.register(path)
```

Comment:
If you omit this line, I think we can drop the explicit dependency on AssetRegistry. WebIO will automatically make this translation if it sees a file path. (I don't know if that's a 100% good idea, but it seemed like it before; let me know if you have any opinions about this!).

Also, if you just register `joinpath(@__DIR__, "..", "deps", "ag-grid")` then you can serve any file under that path (this is useful if your files import other files under this directory) this does require AssetRegistry dependency though.

However, I'm fine with merging this as it is.

Reply:
Alright, I've removed the dependency on AssetRegistry now. It's probably safer to have it, but dunno.